test/model/test/common/wiredtiger_util.cpp

```diff
@@ -422,6 +425,35 @@ wt_ckpt_create(WT_SESSION *session, const char *ckpt_name)
     testutil_check(session->checkpoint(session, config.c_str()));
 }
 
+/*
```

Comment:
Can we create private helper functions for `wt_[get|set]_[stable|oldest]_timestamp`? For example:
```
wt_set_timestamp(WT_CONNECTION *conn, const std::string& timestamp_name, model::timestamp_t timestamp)
{
    char buf[64];

    testutil_snprintf(buf, sizeof(buf), "%s=%" PRIx64, timestamp_name, timestamp);
    testutil_check(conn->set_timestamp(conn, buf));
}
```
What do you think?

Reply:
Good idea—fixed in [cc5b2fb](https://github.com/wiredtiger/wiredtiger/pull/10427/commits/cc5b2fb1f8ed4c230e66945e028a36edeca3fab4).